internal/servers/worker/package_registration.go

```diff
@@ -0,0 +1,5 @@
+package worker
+
+import (
+	_ "github.com/hashicorp/boundary/internal/servers/worker/proxy/tcp"
```

Comment:
I need to side effects import here to trigger the tcp proxy init.  I am open to discussing where this belongs, I figured worker package is best location because then it only builds if we build a worker...

Reply:
LGTM